crates/sui-package-dump/src/query.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use cynic::Operation;
+use cynic::QueryBuilder;
+
+#[cynic::schema("sui")]
+mod schema {}
+
+/// Query types related to GraphQL service limits.
+pub(crate) mod limits {
+    use super::*;
+
+    pub(crate) fn build() -> Operation<Query> {
+        Query::build(())
+    }
+
+    #[derive(cynic::QueryFragment, Debug)]
+    pub(crate) struct Query {
+        pub(crate) service_config: ServiceConfig,
+    }
+
+    #[derive(cynic::QueryFragment, Debug)]
+    pub(crate) struct ServiceConfig {
+        pub(crate) max_page_size: i32,
```

Comment:
`cynic` (the GraphQL client library that I'm using here) maps `Int` to `i32` -- this is technically correct, [according to the spec](https://spec.graphql.org/June2018/#sec-Int). We use it to represent values that are less than 2^53 because that's what we can generally rely on being transferred faithfully by JSON across languages, and `async-graphql` lets us do this as well (maps `u64` to `Int`).

This makes me think that we need to introduce a new scalar, like `UInt64` (...or `UInt53`, or `UInt`?) that maps to a JSON numeric literal but that we can give this meaning. cc @hayes-mysten as our resident expert -- this is not something that we need for JS' sake, but other strongly typed languages are likely to face this issue, and the overhead of this extra scalar seems minimal to accommodate that.